### PR TITLE
OTLP translate: keep identifying attributes in target_info

### PR DESCRIPTION
**Pull request by krajorama**
There is an expectation from OTLP users to see the identifying attributes they send in `target_info`.

```
service.name
service.namespace
service.instance.id
```

However these values are only put into `job` and `instance` currently in `target_info`. I propose to keep them *also* in target info as themselves.

The main reason is UX.

### CODE REVIEW

**Review by aknuds1 — COMMENTED** [review]
Should this perhaps be configurable, for backwards compatibility? We don't know yet whether everyone wants this changed behaviour.

**Comment by krajorama** [comment]
> Should this perhaps be configurable, for backwards compatibility? We don't know yet whether everyone wants this changed behaviour.

Adding labels is backwards compatible in most cases. Unless someone wrote selectors like `{service_name!~".+"}` which seems unlikely. I don't see the value in adding configuration.

**Comment by krajorama** [comment]
setting to draft and add configuration option

**Comment by krajorama** [comment]
Setting to read for review.

File: storage/remote/otlptranslator/prometheusremotewrite/metrics_to_prw_test.go [hunk 1]
```diff
@@ -28,41 +28,73 @@ import (
...
-				for k := 0; k < metricSlice.Len(); k++ {
-					metric := metricSlice.At(k)
-					promName := prometheustranslator.BuildCompliantName(metric, "", false, false)
-					expMetadata = append(expMetadata, prompb.MetricMetadata{
-						Type:             otelMetricTypeToPromMetricType(metric),
-						MetricFamilyName: promName,
-						Help:             metric.Description(),
-						Unit:             metric.Unit(),
-					})
+
```
Comment (aknuds1) [inline storage/remote/otlptranslator/prometheusremotewrite/metrics_to_prw_test.go:37]: [Nit] I'd prefer if you avoid adding this blank line at the function's beginning :)
```suggestion
```

**Review by aknuds1 — DISMISSED** [review]
LGTM, modulo nit.

**Review by jesusvazquez — APPROVED** [review]
As per our conversation in CNCF's slack thread LGTM https://cloud-native.slack.com/archives/C01LSCJBXDZ/p1732539385372279

**Comment by jesusvazquez** [comment]
@krajorama if you could go the extra mile, please update this guide with this flag once its released https://github.com/prometheus/docs/blob/main/content/docs/guides/opentelemetry.md

**Pull request merged**